indy_node/test/context/helper.py

```diff
@@ -0,0 +1,5393 @@
+
+
```

Comment:
Do we have integration tests trying to write these Context examples?

Reply:
These examples are used in `test_send_context.py` and `test_send_get_context.py`